README.md

```diff
@@ -30,11 +30,16 @@ npm run start
 
 ## Configuration
 
-In order to set up your url you need to create `.env` file in the root:
+In order to set up your application you need to create `.env` file in the root:
 
 ```javascript
 /**
  * .env
  */
 REACT_APP_BASE_URL='http://your-url.com'
+REACT_APP_CLIENT_ID='your-client-id'
```

Comment:
Wouldn't it make more sense to create a .env.example file as a template? That's kind of a convention.

Reply:
Done 👍